charts/beyla/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: beyla
-version: 1.6.3
-appVersion: 1.9.8
+version: 1.6.4
```

Comment:
I think if we don't set the version to 2.0, should be at least 1.7, no?

Reply:
I was not sure what was the criteria to set here, as the update of the version looks minimal (no new functionality to the helm chart), but I'm fine updating it to 1.7.0 if you prefer to do it more explicit.

Reply:
I guess normally would be if the chart has breaking changes. Since the config has some, I think it's worth to bump at least to 1.7